lib/evmone/instructions_opcodes.hpp

```diff
@@ -163,6 +163,11 @@ enum Opcode : uint8_t
     OP_DUPN = 0xb5,
     OP_SWAPN = 0xb6,
 
+    OP_DATALOAD = 0xb7,
```

Comment:
Can we reorder this so that `DATALOAD` + `DATALOADI` are next to eachother?

I think you chose this to match the order of `CALLDATA*` and to not having renumber in case `DATALOADI` is rejected.

Let's be optmistic however.